packages/react-data-grid/src/utils/viewportUtils.ts

```diff
@@ -1,40 +1,9 @@
-import { isFrozen } from '../ColumnUtils';
 import { SCROLL_DIRECTION } from '../common/enums';
-import { CalculatedColumn, ColumnMetrics } from '../common/types';
-
-export const OVERSCAN_ROWS = 2;
+import { CalculatedColumn, ScrollPosition, ColumnMetrics } from '../common/types';
 
 const { min, max, ceil, round } = Math;
```

Comment:
Just FYI, this seems weird though it works. For those static functions, I generally would prefer using them the prefix. It's easy to differentiate what is in side the code base and standard lib.

Reply:
I have change it to use `Math.fn`